pontus/week4/app/message.jsx

```diff
@@ -0,0 +1,64 @@
+let React = require('react');
+let MenuItem = require('./menu-item');
+let MessageActions = require('./message-actions');
+
+let ListStore = require("./list-store"); 
+
+
+
+let Message = React.createClass({ 
+	propTypes: { 
+		listID: React.PropTypes.number,
+		messageID: React.PropTypes.number,
+		listName: React.PropTypes.string,
+		isArchived: React.PropTypes.bool,
+		listProperties: React.PropTypes.arrayOf(React.PropTypes.object)
+	},
+
+	moveMessage() {
```

Comment:
I noticed from the flux examples that methods that call an action follow the `_something()` convention, probably to avoid name clashing (`onChange`, `onClick`, etc.). I wonder if this is known and if we should inherit this convention as well. @hulvei3 @calinilie suggestions on this?